checks/xor_check/xor_check.py

```diff
@@ -44,7 +45,8 @@ def gds_xor_check(input_directory, output_directory, magicrc_file_path, gds_gold
 
         # TODO: Try to pass the MAGTYPE as a commandline argument
         os.environ['MAGTYPE'] = 'mag'
-
+        os.environ['PDK_ROOT'] = str(pdk_root.absolute())
```

Comment:
I think we want to call `.resolve()` here, since we can't exclude `open_pdks` shipping symlinks.